### PR TITLE
Add IME_ACTION_GO listener on Password EditText in AuthenticationActivity

### PR DESCRIPTION
**Pull request by adithya321**
Clicking Go in keyboard after entering the password will call "SIGN IN" button's onClick listener.
Previously, it hid the keyboard and then the user has to click the "SIGN IN" button.

### CODE REVIEW

File: src/main/java/org/amahi/anywhere/activity/AuthenticationActivity.java [hunk 1]
```diff
@@ -52,248 +54,258 @@
...
+
+    private void setUpAuthenticationListeners() {
+        setUpAuthenticationTextListener();
+        setUpAuthenticationActionListener();
+    }
+
+    private void setUpAuthenticationTextListener() {
+        getUsernameEdit().addTextChangedListener(this);
+        getPasswordEdit().addTextChangedListener(this);
+        getPasswordEdit().setOnEditorActionListener(new EditText.OnEditorActionListener() {
```
Comment (adithya321) [inline src/main/java/org/amahi/anywhere/activity/AuthenticationActivity.java]: The line which adds IME_ACTION_GO listener

File: src/main/res/layout/layout_authentication.xml [hunk 2]
```diff
@@ -19,100 +18,101 @@
...
-			android:layout_height="wrap_content"/>
-
-	</ViewAnimator>
+        app:passwordToggleEnabled="true">
+
+        <android.support.design.widget.TextInputEditText
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/hint_password"
+            android:imeOptions="actionGo"
```
Comment (adithya321) [inline src/main/res/layout/layout_authentication.xml]: Set imeOption as "actionGo"

File: src/main/res/layout/layout_authentication.xml [hunk 3]
```diff
@@ -19,100 +18,101 @@
...
+    <Space
+        android:layout_width="match_parent"
+        android:layout_height="8dp" />
+
+    <EditText
+        android:id="@+id/edit_username"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/hint_username"
+        android:inputType="text"
```
Comment (adithya321) [inline src/main/res/layout/layout_authentication.xml]: maxLines="1" doesn't work without inputType

File: src/main/res/layout-land/layout_authentication.xml [hunk 4]
```diff
@@ -52,7 +52,8 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:hint="@string/hint_username"
-            android:singleLine="true" />
+            android:inputType="text"
+            android:maxLines="1" />
```
Comment (adithya321) [inline src/main/res/layout-land/layout_authentication.xml]: set maxLines="1" in layout-land

**Comment by adithya321** [comment]
It looks like I have changed a lot since I ran Reformat Code on the files. 
So I have commented on the lines that I changed.

**Comment by cpg** [comment]
Please do NOT submit changes that include reflow/reindent with code changes. Submit those *separately*. We will ask someone to review this PR.

**Comment by adithya321** [comment]
I have opened a new Pull Request #114 without reformatting the code.

**Pull request closed without merging**